### PR TITLE
Add information for sponsors

### PR DESCRIPTION
**Pull request by geokala**

### CODE REVIEW

File: src/information-for-sponsors.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+<div class="box box_blue">
+  <h3>Materials</h3>
+
+  <p>
+    You may arrange to have materials delivered to the venue in advance. Please address all consignments as follows:
+  </p>
+
+  <p>
+    <strong>PyCon UK 2019</strong><br />
```
Comment (AnnJBarr) [inline src/information-for-sponsors.md]: Could we change this to say PyConUK 2022

**Comment by AnnJBarr** [comment]
Apart from updating the year in the postal address LGTM thanks

File: src/information-for-sponsors.md (hunk 1 above)
Comment (geokala) [reply src/information-for-sponsors.md]: Hah, whoops, yes, we can do that.

**Comment by geokala** [comment]
Postal address updated. I'll merge now, and if there are any other issues let me know and new PRs can come to be.

**Pull request merged**